packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/MarkerBuilder.java

```diff
@@ -115,6 +120,13 @@ public void setZIndex(float zIndex) {
     markerOptions.zIndex(zIndex);
   }
 
+  @Override
+  public void setCollisionBehavior(@AdvancedMarkerOptions.CollisionBehavior int collisionBehavior) {
```

Comment:
I think it may be nice to add `setCollisionBehavior` tests for the two implementation in this PR.